README.rst

```diff
@@ -1,5 +1,35 @@
-Installation of Unicore CMS
-===========================
+A4W Style Guide
+================================
+
+For a general overview of Unicore's WoW, see our `WoW Documentation <http://ways-of-working.readthedocs.org/en/latest/process.html#our-front-end-development-process>`_. Overall, we use an approach known as Progressive Enhancement.
+
+HTML
+----
+
+We use Jinja2_ for our templating language. We use an `atomic-design`_-like pattern, so each small section of html code is stored as atoms in ``atoms.html``, larger sections as molecules in ``molecules.html`` and so on, pulling it all together to create each page as a ``.jinja`` file. An example of how this modularised system is implemented can be found here_.
```

Comment:
It's better to use descriptive link text than "here." How about "in the template directory of the unicore-cms repository"?